frameworks/web-ui/react/src/App.tsx

```diff
@@ -1,33 +1,59 @@
 import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
 import './App.css'
+import { Button } from './Button'
+
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [equation, setEquation] = useState<string>("0");
+
+  const addToEquation = (token: string) => {
+    setEquation((equation) => {
+      if (equation == "0") {
```

Comment:
Just as a JavaScript/TypeScript best practice, try avoiding loose equalities. Do the true equality with ===.
Dumb, I know. But it prevents us from stuff like `[] == 0` and `'0' == 0`.